Assets/SteamSample/SteamManager.cs

```diff
@@ -179,14 +193,26 @@ public void HostGame(bool withLobby)
 
             this.hostWithLobby = withLobby;
 
-            Debug.Log($"Hosting game with SteamID #{SteamClient.SteamId}");
+            logger.Info($"Hosting game with SteamID #{SteamClient.SteamId}");
 
             // Make sure we are not already hosting or joining a game
             if (bridge.IsConnected || bridge.IsConnecting)
             {
                 throw new Exception("Failed to host game, CoherenceBride is already connected.");
             }
 
+            try
+            {
+                // Init SteamServer
+                var serverInit = new SteamServerInit(Application.productName, Application.productName);
+                SteamServer.Init(SteamClient.AppId, serverInit, false);
+                logger.Info($"SteamServer initialized");
+            } catch (Exception e)
+            {
+                logger.Error(e.ToString());
```

Comment:
Should this `throw` instead of just logging? - similar to the `bridge` condition above.